src/components/form/form-actions.spec.js

```diff
@@ -161,6 +166,14 @@ describe('FormActions [Component]', () => {
 
         expect(component.find(classname)).to.be.present();
       });
+
+      it('disables the connect button', () => {
+        const connectButton = component
+          .find('button[name="connect"]')
+          .simulate('click');
+
+        expect(connectButton.hasClass('disabled')).to.equal(true);
+      });
```

Comment:
The test that checks if the button is disabled when a syntax error is present.

Reply:
ssdw

Reply:
lol sorry that was Nori ^. I guess she approves too